pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -843,6 +850,60 @@ func (i *Gatherer) gatherNamespaceEvents(namespace string) ([]record.Record, []e
 	return []record.Record{{Name: fmt.Sprintf("events/%s", namespace), Item: EventAnonymizer{&compactedEvents}}}, nil
 }
 
+// GatherServiceAccounts collects ServiceAccount stats
+// from kubernetes default and namespaces starting with openshft.
+//
+// The Kubernetes api https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/serviceaccount.go#L83
+// Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#serviceaccount-v1-core
+//
+// Location of serviceaccounts in archive: config/serviceaccounts
+// See: docs/insights-archive-sample/config/serviceaccounts
+func GatherServiceAccounts(i *Gatherer) func() ([]record.Record, []error) {
+	return func() ([]record.Record, []error) {
+		config, err := i.coreClient.Namespaces().List(i.ctx, metav1.ListOptions{Limit: maxServiceAccountNamespacesLimit})
+		if errors.IsNotFound(err) {
+			return nil, nil
+		}
+		if err != nil {
+			return nil, []error{err}
+		}
+		totalServiceAccounts := 0
+		serviceAccounts := make([]corev1.ServiceAccount, 0)
+		records := make([]record.Record, 0)
```

Comment:
Maybe super dumb question:
Here you are creating an empty `record.Record`, wouldn't it look simpler if you just simply said `var records []record.Record` which does the same thing ?

Reply:
@0sewa0 check this: https://stackoverflow.com/questions/25543520/declare-slice-or-make-slice

Reply:
You are right, the make with 0 doesn't make much sense, it equal to records := []record.Record{}